### ISSUE TITLE
Line wrapping in examples

### ISSUE THREAD

**Issue by mattgarrish** [w3c/respec#3850]
Another request from the epub accessibility horizontal review was to enable line wrapping in markup examples to meet the reflow requirement of WCAG 2.1.

I can again do this manually using `break-spaces` or `pre-wrap`, but wondering if this should be part of the default styling for the `<pre><code>` blocks? (Or maybe it could get applied when the page gets zoomed?)

**Comment by sidvishnoi** [comment]
I agree this should be part of default styling so all specs benefit from it. So this needs to be discussed in [tr-design](https://github.com/w3c/tr-design/). I think we also need to reduce some margins and paddings on small screens and high zoom values so more text can be shown in a single line.

For reference: [Reflow in WCAG 2.1](https://www.w3.org/TR/WCAG21/#reflow)
